roadmap-2019.md

```diff
@@ -76,3 +70,17 @@ Get Pipelines into a reliable release cadence, with trustworthy integration
 tests and performance metrics to guard against regressions. The Pipelines team
 itself should dogfood Pipelines for our own CI and CD infrastructure, alongside
 or instead of Prow, by having Prow use Pipelines somehow.
+
+## Migrate to Pipelines
```

Comment:
I would remove this. Migrating Build to Pipelines could be an item on the Build roadmap, but I don't think it makes sense on Pipelines' roadmap. Whether/when to base Build on Pipelines is not even a settled question at this time, so we shouldn't assume we'll do it just yet.

It's possible we could phrase this as "be stable enough to support Knative Build's implementation" but I think that's already met and exceeded by the 1.0 release section above.

Reply:
kk sounds good, will remove!